### DISCUSSION TITLE
Clock frequency counter/output

### DISCUSSION THREAD

**Discussion by Igordh** [logisim-evolution/logisim-evolution#2049] [Ideas]
I think that a component that outputs the current clock fequency would be nice. Say you have a counter that increases every cycle, you could divide it by the current clock frequency to get the time passed. This would allow for actual wait functions when designing computers, or to add time limits for games made for said computers.

**Comment by maehne** [comment]
Having such a component for synthesis might make sense to get similar results across different development boards with different clock frequencies. In simulation however, the performance is bounded by the frequent redrawing of the circuits, see issue #786. Therefore, the actual speed often deviates considerably from the set target clock speed.
